src/main/java/io/jenkins/plugins/folderauth/roles/FolderRole.java

```diff
@@ -7,8 +7,9 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.TreeSet;
 
-public class FolderRole extends AbstractRole implements Comparable<FolderRole> {
+public class FolderRole extends AbstractRole {
```

Comment:
This changes the signature of the class and deletes the `compareTo(FolderRole)` but `FolderRole` is still a `Comparable` because now `AbstractRole` implements `Comparable` instead.